src/models/tasks.model.ts

```diff
@@ -0,0 +1,61 @@
+import { AllowNull, Column, DataType, Model, PrimaryKey, Table } from 'sequelize-typescript';
+import { DataTypes } from 'sequelize/types';
+
+@Table({ modelName: 'task', timestamps: true, paranoid: true })
+export default class Task extends Model<Task> {
+  @PrimaryKey
+  @Column({
+    defaultValue: DataType.UUIDV4,
+    type: DataType.UUID,
+  })
+  taskId: string;
+
+  // @ForeignKey
+  // @Column({
+  //   defaultValue: DataType.UUIDV4,
+  //   type: DataType.UUID,
+  // })
+  // userId: string;
+
+  @AllowNull(false)
+  @Column(DataType.STRING(40))
+  title: string;
+
+  @AllowNull(false)
+  @Column(DataType.DECIMAL(1, 2))
+  UserWeight: number;
```

Comment:
why are some fields uppercase but others lowercase? ex) taskId, title, and UserWeight?

Reply:
Because I didn't proofread this before I pushed haha. Let me fix that

Reply:
Changed, I'll let you resolve

Reply:
Not given an option to resolve.  Maybe @jin0s add me as collaborator?

Reply:
my bad, I added your email instead of your github account